tsconfig.json

```diff
@@ -5,7 +5,9 @@
     "outDir": "./lib",
     "target": "ESNext",
     "moduleResolution": "node",
-    "module": "CommonJS"
+    "module": "CommonJS",
+    "declaration": true,
+    "declarationMap": true
```

Comment:
API Extractor works off the `.d.ts` files so we need to generate them now.

Reply:
ack